### PR TITLE
fix/style-css

### PR DESCRIPTION
**Pull request by yashuk803**

### CODE REVIEW

**Comment by yashuk803** [comment]
https://trello.com/c/BaBqqcCc/156-add-white-background-for-the-half-of-page-master-of-adding-website
https://trello.com/c/6ejEcpId/159-add-paddings-to-the-main-layout-according-to-design

File: frontend/src/scss/main.scss [hunk 1]
```diff
@@ -56,4 +56,17 @@ svg path {
 
 .v-btn {
     text-decoration: none !important;
-}
\ No newline at end of file
+}
+
+.content-with-padding {
+    padding: 70px 66px 0 80px;
```
Comment (lenchvolodymyr) [inline frontend/src/scss/main.scss]: what are the magic numbers?) And why is bottom without padding?

Comment (lenchvolodymyr) [reply frontend/src/scss/main.scss]: There is really small distance from header to content.

Comment (lenchvolodymyr) [reply frontend/src/scss/main.scss]: And these styles are probably better

width: 90%;
    margin: 0 auto;
    padding: 32px 0;

**Pull request merged**